models/channel/src/channel-draft/presets/hub.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { TChannelDraft, TChannelDraftGraphql } from '../../types';
+import hub from './hub';
+
+describe('hub channel', () => {
+  it('should match REST snapshot', () => {
+    const channel = hub().build<TChannelDraft>();
+
+    expect(channel).toMatchInlineSnapshot(`
+      {
+        "address": undefined,
+        "custom": undefined,
+        "description": {
+          "de": undefined,
+          "en": undefined,
+          "en-US": "Hub",
+          "fr": undefined,
+        },
+        "geoLocation": undefined,
+        "key": "hub",
+        "name": {
+          "de": undefined,
+          "en": undefined,
+          "en-US": "Hub",
+          "fr": undefined,
+        },
+        "roles": [
+          "ProductDistribution",
+          "InventorySupply",
+        ],
+      }
+    `);
+  });
+
+  it('should match graphql snapshot', () => {
+    const channel = hub().buildGraphql<TChannelDraftGraphql>();
+
+    expect(channel).toMatchInlineSnapshot(`
+      {
+        "__typename": "ChannelDraft",
+        "address": undefined,
+        "custom": undefined,
```

Comment:
I wonder if we'll need some sort of prefiltering to clear these `undefined` values when we make a query. Right now we sort of generate _everything_.